src/main/java/ListOfTasks.java

```diff
@@ -0,0 +1,130 @@
+import java.util.ArrayList;
+
+public class ListOfTasks {
+    private static int count;
```

Comment:
Is the name count clear enough? The readers might be confused what count you want to represent here!